frontend/packages/employee-frontend/src/components/shared/atoms/buttons/Button.tsx

```diff
@@ -69,13 +76,51 @@ export const StyledButton = styled.button`
   ${defaultButtonTextStyle}
 `
 
+const UnderRow = styled.div`
+  padding: 0 12px;
+  margin-top: ${DefaultMargins.xxs};
+  margin-bottom: -20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  position: absolute;
+
+  font-size: 12px;
+  text-overflow: ellipsis;
+
+  word-wrap:break-word;
+  width: 120px;
```

Comment:
Would have preferred a responsive width, but couldn't come up with a neat trick for making this as wide as the button above. This looked ok in the inspector mode mobile simulator, though.